skyportal/models.py

```diff
@@ -1758,6 +1758,65 @@ def api_class(self):
     def listener_class(self):
         return getattr(facility_apis, self.listener_classname)
 
+    fields = relationship("InstrumentField")
+    tiles = relationship("InstrumentFieldTile")
+
+
+class InstrumentField(ha.Region, Base):
+    """A multi-order healpix representation of a telescope's field shape,
+    as represented by many InstrumentFieldTiles."""
+
+    tile_class = lambda: InstrumentFieldTile  # pylint: disable=E731 # noqa
+
+    instrument_id = sa.Column(
+        sa.ForeignKey('instruments.id', ondelete="CASCADE"),
+        nullable=False,
+        doc='Instrument ID',
+    )
+
+    instrument = relationship(
+        "Instrument",
+        foreign_keys=instrument_id,
+        doc="The Instrument that this field belongs to",
+    )
+
+    field_id = sa.Column(
```

Comment:
How is this different from the `id` column this class inherits from `Base`?

Reply:
It shouldn't have to be unique (two telescopes will have a field 1)... is there a unique requirement in Base?

Reply:
Ah, okay, perhaps make that more explicit in the column desc. Thanks!

Reply:
Yes added!